docs/Troubleshooting.md

```diff
@@ -169,9 +171,49 @@ This is a known issue an occurs on first upload of Linux data into an OMS worksp
 * Connection to the OMS Service is blocked
 * OMS Agent for Linux data is backed up
 
-### Resolutions
+#### Resolutions
 * Check if onboarding the OMS Service was successful by checking if the following file exists: `/etc/opt/microsoft/omsagent/conf/omsadmin.conf`
  * Re-onboard using the omsadmin.sh command line [instructions](https://github.com/Microsoft/OMS-Agent-for-Linux/blob/master/docs/OMS-Agent-for-Linux.md#onboarding-using-the-command-line)
 * If using a proxy, check proxy troubleshooting steps aboce
 * In some cases, when the OMS Agent for Linux cannot talk to the OMS Service, data on the Agent is backed up to the full buffer size: 50 MB. The OMS Agent for Linux should be restarted by running the following command `service omsagent restart` or `systemctl restart omsagent`.
  * **Note:** This issue is fixed in Agent version >= 1.1.0-28
+
+### My portal side configuration for (Syslog/Linux Performance Counter) is not being applied
+#### Probable Causes
+* The OMS Agent for Linux Configuration Agent has not picked up the latest portal side configuration
+* The changed settings in the portal were not applied
+
+#### Resolutions
+
+**Background:** `omsconfig` is the OMS Agent for Linux configuration agent that looks for new portal side configuration every 5 minutes. This configuration is then applied to the OMS Agent for Linux configuration files located at /etc/opt/microsoft/omsagent/conf/omsagent.conf. 
+
+* In some cases the OMS Agent for Linux configuration agent might not be able to communicate with the portal configuration service resulting in latest configuration not being applied.
+ * Check that the `omsconfig` agent is installed
+  * `dpkg --list omsconfig` or `rpm -qi omsconfig`
+  * If not installed, reinstall the latest version of the OMS Agent for Linux
+
+* Check that the `omsconfig` agent can communicate with the OMS Portal Service
+  * Run the following command `sudo su omsagent -c 'python /opt/microsoft/omsconfig/Scripts/GetDscConfiguration.py'`
```

Comment:
Also run this command if the GetDscConfiguration.py doesn't return anything.
sudo su omsagent -c /opt/microsoft/omsconfig/Scripts/PerformRequiredConfigurationChecks.py
It will talk to the DSC service to get the configuration, and apply locally.